projects/packages/my-jetpack/_inc/components/product-detail-card/style.module.scss

```diff
@@ -1,26 +1,37 @@
 .card {
-	background-color: var(--jp-white);
+	background-color: var( --jp-white );
 	height: 100%;
+	padding: calc( var( --spacing-base ) * 6 );
+	position: relative;
+
+	.container {
```

Comment:
This doesn't need to be included inside the `.card` class, IMO it outside improves the readability.

Reply:
let me move it out.

Reply:
https://github.com/Automattic/jetpack/pull/23743/commits/0562c03f450750d93d2520ab664fc1d2c920889d